content/blog/2023/01/release-1_5_0.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Kubewarden 1.5.0 release
+authors:
+- José Guilherme Vanz
+date: 2023-01-18
+---
+
+Today we're pleased to announce the availability of Kubewarden 1.5.0!
+
+Among all the dependencies updates and minor fixes in all the Kubewarden components it's worth
+to mention some of them and a new interesting feature.
+
+## Policy evaluation timeout
+
+In the continuous effort of the Kubewarden team to keep the project safer on every release,
+policy server now allows users to define how much time a policy can spend to evaluate an admission request.
+This security feature, which is enabled by default, ensures that infinite loops and other scenarios
+where policies may get stuck will not consume all the resources available turning policy server more
+reliable to Denial of Service (DOS) attacks.
```

Comment:
```suggestion
The Kubewarden team is constantly working to improve the security posture of the project. As part of this efforts,
we're excited to introduce the new "policy evaluation timeout" feature.

Starting from this release, Policy Server will interrupt the evaluation of admission requests after a certain amount of time has elapsed.
This security feature, which is enabled by default, prevents a Policy Server from running out of computing
resources because one or more of its policy evaluations are stuck in infinite loops.
This mitigates a type of Denial Of Service (DOS) attacks against the Policy Server. 
```